packages/react-vapor/src/components/modal/ConfirmationModalProvider.tsx

```diff
@@ -19,37 +23,47 @@ export interface IConfirmationModalProviderProps {
     confirmButtonText?: string;
 }
 
-export const ConfirmationModalProvider: React.FunctionComponent<IConfirmationModalProviderProps> = ({
+const enchance = connect(null, (dispatch: IDispatch, ownProps: IConfirmationModalProviderProps) => ({
```

Comment:
```suggestion
const enhance = connect(null, (dispatch: IDispatch, ownProps: IConfirmationModalProviderProps) => ({
```
hehe, small typo here :)